src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -112,15 +112,29 @@ export const monoPlatform: Platform = {
   },
 };
 
-function addScriptTagsToDocument() {
+function addScriptTagsToDocument(resourceLoader: WebAssemblyResourceLoader) {
   const browserSupportsNativeWebAssembly = typeof WebAssembly !== 'undefined' && WebAssembly.validate;
   if (!browserSupportsNativeWebAssembly) {
     throw new Error('This browser does not support WebAssembly.');
   }
 
+  // The dotnet.*.js file has a version or hash in its name as a form of cache-busting. This is needed
+  // because it's the only part of the loading process that can't use cache:'no-cache' (because it's
+  // not a 'fetch') and isn't controllable by the developer (so they can't put in their own cache-busting
+  // querystring). So, to find out the exact URL we have to search the boot manifest.
+  const dotnetJsResourceName = Object
+    .keys(resourceLoader.bootConfig.resources.runtime)
+    .filter(n => n.startsWith('dotnet.') && n.endsWith('.js'))[0];
```

Comment:
Can this conflicts with a user resource (or is the `runtime` section special)?

Reply:
Everything in `blazor.boot.json` comes from the framework - there are no user-supplied files there.